src/ydata_synthetic/synthesizers/loss.py

```diff
@@ -1,53 +1,70 @@
-from tensorflow import random
-from tensorflow import reshape, shape, math, GradientTape, reduce_mean
-from tensorflow import norm as tfnorm
+import tensorflow as tf
```

Comment:
Can we revert to what was done previous, i.e. importing only what is necessary,

Reply:
Done.